experimental/add_to_app/android_fullscreen/app/src/main/java/dev/flutter/example/androidfullscreen/MyApplication.kt

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package dev.flutter.example.androidfullscreen
+
+import android.app.Application
+import io.flutter.embedding.engine.FlutterEngine
+import io.flutter.embedding.engine.FlutterEngineCache
+import io.flutter.embedding.engine.dart.DartExecutor
+import io.flutter.plugin.common.MethodChannel
+import io.flutter.view.FlutterMain
+
+const val ENGINE_ID = "1"
+
+class MyApplication : Application() {
+    var count = 0
+
+    private var channel: MethodChannel? = null
+
+    override fun onCreate() {
+        super.onCreate()
+
+        FlutterMain.startInitialization(this)
```

Comment:
FYI, these 2 calls will no longer be required once I merge https://github.com/flutter/engine/pull/12806

Reply:
Thanks for the reminder. Just CCed myself on your PR.